src/_guides/testing.md

```diff
@@ -130,14 +130,11 @@ For details, see the
 
 Consider using continuous integration (CI) to build your project
 and run its tests after every commit. Two CI services for GitHub are
-[Travis CI](https://travis-ci.org/) (for OS X and Unix) and
+[Github Actions](https://github.com/features/actions) and
 [AppVeyor](https://www.appveyor.com/) (for Windows).
 
-Travis has built-in support for Dart projects.
-Learn more at the following links:
+Learn more abou GitHub Actions at the following link:
+
+* [Testing Dart packages with GitHub Actions](https://medium.com/flutter-community/testing-dart-packages-with-github-actions-4c2c671b1e34)
```

Comment:
Not a hard and fast rule, but we usually don't have bulleted lists with a single item. Are there any other good resources? Also, since this is reposted from a non-medium source — and the author says the original is better on mobile — I thought we might want to use that original URL.

```suggestion
* [Testing Dart packages with GitHub Actions](https://poetryincode.dev/testing-dart-packages-with-github-actions)
```

Reply:
Should I add these also : [https://medium.com/mobile-development-group/github-actions-for-flutter-cf02923d7b5d](Github Actions for Flutter) and [https://medium.com/better-programming/ci-cd-for-flutter-apps-using-github-actions-b833f8f7aac](CI/CD for Flutter Apps Using GitHub Actions)

Reply:
Should I add these also [CI/CD for Flutter Apps Using GitHub Actions](https://medium.com/better-programming/ci-cd-for-flutter-apps-using-github-actions-b833f8f7aac) and [Github Actions for Flutter](https://medium.com/mobile-development-group/github-actions-for-flutter-cf02923d7b5d)

Reply:
Maybe the first one (I don't know enough to judge). The second one is premium, so let's not point to that.

Instead of more articles, how about we link to some PRs/code related to GitHub Actions support? A lot of Dart team packages have switched from Travis to GHA lately.

Reply:
If you look at the profile of @athomas, you'll find a lot of PRs switching from Travis to GitHub Actions.

Reply:
This is a good recent example without a lot of the tinkering potentially causing confusion - https://github.com/dart-lang/markdown/pull/353

Plus there's some comments

Reply:
So, should I remove all the articles and add link to this [dart-lang/markdown#353](https://github.com/dart-lang/markdown/pull/353) ?

Reply:
How about we keep the most helpful non-premium article and point to the markdown repo/PR?

Reply:
I'm not sure what to write about the markdown repo.

Reply:
I've edited the file to have some wording that might work. What do you think?

Reply:
I think its great. Do I have to pull these to my branch and then push again? because I made some commits earlier.

Reply:
Nope, I committed to your branch (figuring you could always undo my changes 😄 ). I'll commit this PR. Thank you!

Reply:
Thanks! This was my first ever accepted contribution!

Reply:
Congratulations, @AliAkberAakash!